### PR TITLE
NP9312-Create-app-client-for-DLR, client created

### PR DESCRIPTION
**Pull request by truhacevkir**
Ser det bra ut?

### CODE REVIEW

File: template.yaml [hunk 1]
```diff
@@ -1277,6 +1279,80 @@ Resources:
...
+        - custom:orgFeideDomain
+        - custom:firstName
+        - custom:lastName
+        - custom:accessRights
+        - custom:customerId
+        - custom:allowedCustomers
+        - custom:roles
+        - custom:nvaUsername
+        - custom:cristinId
+        - custom:topOrgCristinId
```
Comment (axthosarouris) [inline template.yaml:1326]: do you need these 3?

File: template.yaml [hunk 2]
```diff
@@ -1277,6 +1279,80 @@ Resources:
...
```
Comment (axthosarouris) [inline template.yaml:1352]: ditto

File: template.yaml (hunk 1 above)
Comment (truhacevkir) [reply template.yaml]: probably not, have removed them.

File: template.yaml (hunk 2 above)
Comment (truhacevkir) [reply template.yaml]: removing these also

**Pull request merged**